velox/docs/functions/spark/decimal.rst

```diff
@@ -18,3 +18,41 @@ Decimal Special Forms
     Create ``decimal`` of requsted precision and scale from an unscaled bigint value ``x``.
     By default, the value of ``nullOnOverflow`` is true, and null will be returned when ``x`` is too large for the result precision.
     Otherwise, exception will be thrown when ``x`` overflows.
+
+.. spark:function:: decimal_round(decimal[, scale]) -> [decimal]
+
+    Returns ``decimal`` rounded to a new scale using HALF_UP rounding mode. In HALF_UP rounding, the digit 5 is rounded up.
+    ``scale`` is the new scale to be rounded to. It is 0 by default, and any integer is allowed to be its value.
```

Comment:
> and any integer is allowed to be its value

It would be nice to clarify how large positive or negative values are processed. It is not obvious.

Reply:
Yes, added that description. Also added unit tests for scale being INT_MIN and INT_MAX.